lib/response.ex

```diff
@@ -71,6 +45,15 @@ defmodule HTTPServer.Response do
   end
 
   defp format_response_headers(headers) do
-    for {key, val} <- headers, into: "", do: "#{key}: #{val}#{@carriage_return}"
+    for {n, v} <- headers,
+        into: "",
+        do: "#{format_header_name(n)}: #{v}#{@carriage_return}"
+  end
+
+  defp format_header_name(header_name) do
+    to_string(header_name)
+    |> String.split("_")
+    |> Enum.map(&String.capitalize/1)
+    |> Enum.join("-")
   end
```

Comment:
Could these move into `Headers` ? should they?

Reply:
OOoh good question. I put them in Response, because I created them to be helper functions for `format_response/1`. But I do think a case can be made that they should be moved into Headers since their only responsibility is formatting the headers! I could also maybe make `collect_headers` into a function that is moved into the Headers module as well.

Reply:
Just moved all of these to Headers, which allowed me to simplify `send_resp/3` as well